scripts/release_info.py

```diff
@@ -31,14 +34,24 @@
 GIT_CMD_SHOW_FORMAT_STRING = 'git show %s:feconf.py'
 ISSUE_URL_FORMAT_STRING = 'https://github.com/oppia/oppia/issues/%s'
 ISSUE_REGEX = re.compile(r'#(\d+)')
+PR_NUMBER_REGEX = re.compile(r'\(#(\d+)\)')
 GROUP_SEP = '\x1D'
 VERSION_RE_FORMAT_STRING = r'%s\s*=\s*(\d+|\.)+'
 FECONF_VAR_NAMES = ['CURRENT_STATES_SCHEMA_VERSION',
                     'CURRENT_COLLECTION_SCHEMA_VERSION']
 FIRST_OPPIA_COMMIT = '6a7138f5f603375e58d1dc3e1c4f1c80a126e249'
+NO_LABEL_CHANGELOG_CATEGORY = 'Uncategorized'
 
 Log = collections.namedtuple('Log', ['sha1', 'author', 'email', 'message'])
 
+_PARSER = argparse.ArgumentParser()
+_PARSER.add_argument(
```

Comment:
Just to check -- do we want to make this optional or not (if it's optional, the script runs but just displays an error message in place of the changelog bits)? Not sure, and I'm happy either way; just wanted to get your thoughts.

(If it's not optional, what happens when you run it without passing in the arg?)

Reply:
I made it optional. So if you run it without, it won't output the categorized changelog. 

However in future, if we move to pyGithub throughout release_info.py (as we discussed last time), then it might be useful to make it required. For now, I have made it optional, and printed a message on how to get a token, and pass it while calling the script.